### DISCUSSION TITLE
Change volume with scroll wheel while hovering over the tray icon

### DISCUSSION THREAD

**Discussion by terax6669** [File-New-Project/EarTrumpet#1040] [Ideas]
This is a feature from xfce that I'm missing on Windows.

**Comment by riverar** [comment]
@terax6669 It already works today, did you try it before posting?

**Comment by terax6669** [comment]
I did try it, but apparently I'm dumb.

There was no visual feedback so I just assumed it didn't work. Maybe that's something that could be improved? I don't know if you can trigger the native version of volume notification somehow but if not then briefly showing a single volume bar like that looks like in the tray popout should do the trick.

Sorry for the confusion.

**Reply by riverar** [reply]
@terax6669 Haha, not dumb 😂 We do have a similar suggestion to show the tooltip as it changes. I suspect that would've helped a lot here.

**Reply by riverar** [reply]
Err. Actually, I think this just broke in a recent Windows 11 Insider Build. Oh no. What has Microsoft done...

**Reply by terax6669** [reply]
Did you have time to take a look at it?

**Reply by terax6669** [reply]
Still nothing, eh?

If we're never getting the native popup then maybe fix the bug with the current tooltip?

<img width="288" height="58" alt="image" src="https://github.com/user-attachments/assets/3da1e765-822d-4dc8-a934-6e6bd7929fbe" />

It updates the tooltip first and then it adjusts the value... so it's always 2% off.

**Reply by riverar** [reply]
I don't believe any of these issues exist in the dev version (e.g., see https://github.com/File-New-Project/EarTrumpet/issues/1483). Can you try that out and verify?
